test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,426 @@
+          smartContract,
+        );
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+        // create token
+        await driver.openNewPage(
+          `http://127.0.0.1:8080/?contract=${contractAddress}`,
+        );
+        let windowHandles = await driver.getAllWindowHandles();
+        const extension = windowHandles[0];
+        await driver.findClickableElement('#deployButton');
+
+        // add token from dapp
+        await driver.clickElement({
+          text: 'Add Token to Wallet',
+          tag: 'button',
+        });
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.clickElement({ text: 'Add token', tag: 'button' });
+        windowHandles = await getWindowHandles(driver, 2);
+        // transfer token from dapp
+        await driver.clickElement({ text: 'Transfer Tokens', tag: 'button' });
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.waitForSelector({ text: '1.5 TST', tag: 'h1' });
+        await driver.clickElement({ text: 'Edit', tag: 'button' });
+
+        // customizes gas
+        await driver.clickElement(
+          { text: 'Edit suggested gas fee', tag: 'button' },
+          10000,
+        );
+        const inputs = await driver.findElements('input[type="number"]');
+        const gasLimitInput = inputs[0];
+        const gasPriceInput = inputs[1];
+        await gasLimitInput.fill('60000');
+        await gasPriceInput.fill('10');
+
+        await driver.clickElement({ text: 'Save', tag: 'button' });
+        await driver.findElement({ tag: 'span', text: '0.0006' });
+        const tokenAmount = await driver.findElement(
+          '.confirm-page-container-summary__title-text',
+        );
+        const tokenAmountText = await tokenAmount.getText();
+        assert.equal(
+          tokenAmountText,
+          '1.5 TST',
+          'Token amount should be 1.5 TST',
+        );
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+        await driver.switchToWindow(extension);
+        // checks if transaction finish correctly from extension
+        await driver.clickElement({ tag: 'button', text: 'Activity' });
+        await driver.waitForSelector({
+          css: '.transaction-list__completed-transactions .transaction-list-item__primary-currency',
+          text: '-1.5 TST',
+        });
+        await driver.waitForSelector({
+          css: '.list-item__heading',
+          text: 'Send TST',
+        });
```

Comment:
We should be able to remove the code block below, as theres already a wait for the completed transaction
```
await driver.waitForSelector({
          css: '.list-item__heading',
          text: 'Send TST',
        });
```

Reply:
Done